agent.exs

```diff
@@ -1,55 +1,62 @@
 defmodule Appsignal.Agent do
   def version, do: "0318770"
 
+  def mirrors do
+    [
+      "https://appsignal-agent-releases.global.ssl.fastly.net",
+      "https://d135dj0rjqvssy.cloudfront.net",
+    ]
+  end
```

Comment:
@luismiramirez have we updated the appsignal-agent's ship task to add these mirrors to the `agent.exs` template?

Reply:
Discussed with team in call. I'll pick up this change.